float32 goal_x
float32 goal_y
float32 goal_angle
float32 base_speed
float32 base_ang_v
float32 pos_tolerance
bool drive_forwards
---
bool success
---
float32 current_x
float32 current_y
